Tools/patchcheck/patchcheck.py

```diff
@@ -175,6 +187,15 @@ def report_modified_files(file_paths):
     'Tools/c-analyzer/cpython/_parser.py',
 })
 
+#: Python files with bad syntax
```

Comment:
```suggestion
# Python files with bad syntax
```

Reply:
Copied the style from the above (this is the style Sphinx uses for documentation comments, but unsure why the above used it)